README.md

```diff
@@ -102,4 +102,46 @@ When you have implemented the two (or more) classes, describe the complexity of
 
 **FIXME: OPERATION COMPLEXITY**
 
+For doubly linked lists: 
+- Accesing is in linear time
+- Searching is in linear time 
+- Adding or deleting elements are in constant time
+
+Function   | ListGenome | LinkedListGenome | LinkedListGenome2
+-----------|------------|------------------|------------------
+init       | O(n)       | O(n)             | O(n)
+insert_te  | O(m+k) + O(disable_te)| O(n+m) + O(disable_te) | O(n+m+k + disable_te)
```

Comment:
The first should be $O(n+m)$ not $O(m+k)$. The second should also have a $+k$--you search through the existing TEs to find the one that should be disabled.